### DISCUSSION TITLE
Welcome to ACS Authentication Server Node.js Sample Discussions!

### DISCUSSION THREAD

**Discussion by lei-msft** [Azure-Samples/communication-services-authentication-hero-nodejs#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community by providing help and receiving feedback. We hope that you:
  * [Share ideas](https://github.com/Azure-Samples/communication-services-authentication-hero-nodejs/discussions/categories/ideas).
  * [Ask questions](https://github.com/Azure-Samples/communication-services-authentication-hero-nodejs/discussions/categories/q-a) you’re wondering about ACS Authentication Server Node.js Sample.
  * [Share what you're building](https://github.com/Azure-Samples/communication-services-authentication-hero-nodejs/discussions/categories/show-and-tell) with the community!
  * Engage with other community members.

  Please remember that this is a community we build together 💪. So please be kind and considerate to everyone in the community!

  To get started, comment below with an introduction of yourself and tell us about what excites you about ACS Authentication Server Sample!

**Comment by xrrlzzq** [comment]
Awesome! It is exact what I need for my coming personal project. Thank you so much for saving me a lot of time.

By the way, I am curious how I incorporate this sample with my coming project which users usually use social accounts to signin, like Facebook, Google, Wechat etc..

**Reply by lei-msft** [reply]
> Awesome! It is exact what I need for my coming personal project. Thank you so much for saving me a lot of time.
> 
> By the way, I am curious how I incorporate this sample with my coming project which users usually use social accounts to signin, like Facebook, Google, Wechat etc..

Hi @xrrlzzq. Thank you so much for your support for Azure Communication Services and ACS Authentication Server Sample. :wink:

About your question. I recommend you use [Azure Active Directory B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/overview) combined with our sample. In addition, we will add a section about how to incorporate social accounts with our project in our documentation soon. Please stay focus on this project. :eyes: I will also keep you informed.

If you have any other questions, please feel free to create [a new discussion](https://github.com/Azure-Samples/communication-services-authentication-hero-nodejs/discussions/new) in :pray: Q&A section. We are so glad that the ACS Authentication Server sample can help you.

Related Resources:

1. [Azure Active Directory B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/overview)
2. [Quickstart: Set up sign in for an ASP.NET application using Azure Active Directory B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/quickstart-web-app-dotnet)
3. [Quickstart: Set up sign in for a single-page app using Azure Active Directory B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/quickstart-single-page-app)
4. [Tutorial: Create an Azure Active Directory B2C tenant](https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-create-tenant)

**Reply by davrous** [reply]
> Awesome! It is exact what I need for my coming personal project. Thank you so much for saving me a lot of time.
> 
> By the way, I am curious how I incorporate this sample with my coming project which users usually use social accounts to signin, like Facebook, Google, Wechat etc..

Hi @xrrlzzq, if interested by social accounts to signit, please have a look to my sample/tutorial: https://github.com/davrous/acsauth

**Comment by ram4git** [comment]
The MinimalClient that is part of the this sample application is not able to add participants to existing call. It appears disabled and show only current participants and do not have any options to other participants though logged in user or application has ManageCall permission. Is there anything extra that needs to done? Any pointers are appreciated.
